lumicks/pylake/adjustments.py

```diff
@@ -80,7 +80,7 @@ def _get_data_rgb(self, image, channel="rgb"):
             image[:, :, missing_channel_index] = 0
 
         if not self.mode:
-            return image / np.max(image)
+            return image / (maximum if (maximum := image.max()) else 1)
```

Comment:
Perhaps:
~~~suggestion
            return image / maximum if maximum := image.max() else image
~~~

Reply:
It needs one more bracket because of precedence, but I do like the suggestion of not dividing at all, thanks! 👍